File: Hackwithinfy/distinctsubstring.java
import java.util.Scanner;

class Main{

    static int countstring(String str)
    {
        int length = str.length();
        int answer = length * (length+1)/2;

        return answer;
    }

    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);

        String str = sc.nextLine();

        System.out.println(countstring(str));
    }
}

File: Leet Code/distinctsubstring.c
#include<stdio.h>
#include<string.h>

int countstring(char *str)
{
    int length = strlen(str);
    int answer = length * (length+1)/2;

    return answer;
}

int main()
{
    char *str;
    scanf("%ms",&str);

    printf("%d", countstring(str));

    return 0;
}
